addons/pos_restaurant/static/src/overrides/components/navbar/navbar.js

```diff
@@ -31,52 +22,14 @@ patch(Navbar.prototype, {
     get showEditPlanButton() {
         return true;
     },
-    setFloatingOrder(floatingOrder) {
-        this.pos.set_order(floatingOrder);
-
-        const props = {};
-        const screenName = floatingOrder.get_screen_data().name;
-        if (screenName === "PaymentScreen") {
-            props.orderUuid = floatingOrder.uuid;
+    get orderName() {
+        const order = this.pos.get_order();
+        if (!order.isDirectSale) {
+            return order.getName();
```

Comment:
```suggestion
        return this.pos.get_order().isDirectSale ? _t("Direct Sale") : this.pos.get_order().getName();
```

Can also be done in XML